src/framework/viewHeader/dateRange/DateRange.jsx

```diff
@@ -155,7 +155,9 @@ DateRange.propTypes = {
   startDate: PropTypes.object,
```

Comment:
As you already would be aware, the tests for the `DateRange` are missing such as if the dates are being set properly, if the state is changing with the events and if the component is being rendered properly with the supposed structure and classes within the DOM. 

This would be a lot and I'd say this PR is okay to go without given the circumstances, but I'd like to do the tests for this component in a separate branch under your guidance (assuring you that it would be minimal) next week or as time permits. Or may be if there is a particular reason we can skip the tests for this, then we can leave as is.